src/lib/network/network_autoyast.rb

```diff
@@ -69,6 +70,10 @@ def create_udevs
       no_rules &&= ex_udev_rules.empty?
       return if no_rules
 
+      # for the purpose of setting the persistent names, create the devices 1st
+      s390_devices = ay_networking_section.fetch("s390-devices", {})
+      s390_devices.each { |rule| LanItems.createS390Device(rule) } if Arch.s390
```

Comment:
IMHO the whole block should be in the `if Arch.s390` condition. Or does it make sense to fetch the "s390-devices" on non-S390? This is a bit strange...

Reply:
well, I'm not author of this change but ... it is expectable that the section is empty on non-s390 ... and if not then it is catched by the second condition.

so, having it encapsulated in a if statement is cleaner, but doesn't hurt when done this way.

Reply:
ok